pkg/rtc/participant.go

```diff
@@ -1307,6 +1314,40 @@ func (p *ParticipantImpl) configureReceiverDTX() {
 	}
 }
 
+func (p *ParticipantImpl) onStreamedTracksChange(paused map[string][]string, resumed map[string][]string) error {
```

Comment:
do we need the map structure? since the protocol doesn't require it.

Reply:
Don't need it. But, did not want to use the protocol types too far deep in code. Is it okay to use `livekit.StreamedTrack` all the way across? I am still thinking of `sfu` bits as a logical separation (even after merging) and trying to generic types.

Reply:
I agree with the separation.. perhaps a struct within the SFU package? this is fine as well.. just that the map population logic seems unnecessary.

Reply:
Sounds good @davidzhao . Will make a struct inside SFU package in an upcoming PR.